pkg/agent/manager/manager_test.go

```diff
@@ -640,7 +640,7 @@ func TestSurvivesCARotation(t *testing.T) {
 		fetchX509SVID: fetchX509SVIDForTestSurvivesCARotation,
 		// Give a low ttl to get expired entries on each synchronization, forcing
 		// the manager to fetch entries from the server.
-		svidTTL: 3,
+		svidTTL: 0,
```

Comment:
#749 will remove this dependency on real time

Reply:
I think given the nature of this change and my previous issue with it, this PR (and probably most/all others) should take a dependency on #749 merging lest we all keep getting inconsistent test failures